org.nickvision.tagger.json

```
@@ -0,0 +1,104 @@
+{
+    "app-id" : "org.nickvision.tagger",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "org.nickvision.tagger",
+    "tags" : [
+        "devel"
+    ],
+    "finish-args":[
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
+        "--filesystem=home",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules" : [
+        {
+            "name": "jsoncpp",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.9.5.tar.gz",
+                    "sha256": "f409856e5920c18d0c2fb85276e24ee607d2a09b5e7d5f0a371368903c275da2"
+                }
+            ]
+        },
+        {
+            "name": "libcurl",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://curl.se/download/curl-7.83.1.tar.gz",
+                    "sha256": "93fb2cd4b880656b4e8589c912a9fd092750166d555166370247f09d18f5d0c0"
+                }
+            ]
+        },
+        {
+            "name": "curlpp",
+            "buildsystem": "cmake",
```

```suggestion
            "buildsystem": "cmake-ninja",
```

it's prefered (unless it doesn't work)